doc/library/config.txt

```diff
@@ -705,6 +705,24 @@ import theano and print the config variable, as in:
     Generate a warning when the destroy_map or view_map tell that an op work
     inplace, but the op did not reuse the input for its output.
 
+.. attribute:: config.NanGuardMode.nan_is_error
+
+    Bool value, default: True
+
+    Do NanGuardMode generate an error when it see a nan.
+
+.. attribute:: config.NanGuardMode.inf_is_error
+
+    Bool value, default: True
+
+    Do NanGuardMode generate an error when it see an inf.
+
+.. attribute:: config.NanGuardMode.nan_is_error
+
+    Bool value, default: True
+
+    Do NanGuardMode generate an error when it see a big value (>1e10).
```

Comment:
For this attribute and the 2 previous, sentence structure is a bit weird. Replace "Do NanGuardMode [...]" with "Whether NanGuardMode should", maybe?